_search-plugins/star-tree-index.md

```diff
@@ -39,26 +39,26 @@ The following image illustrates a standard star-tree index structure.
+Each node in a star-tree index points to a range of star-tree documents. Nodes can be further split into child nodes based on the [max_leaf_docs configuration]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/star-tree/#stix-configuration-options). The number of documents that a leaf node points to is less than or equal to the value set in `max_leaf_docs`. This ensures that the maximum number of documents that need to traverse nodes to derive an aggregated value is at most the number of `max_leaf_docs`, which provides predictable latency.
 
 ### Star nodes
 
-Star nodes are children of non-leaf nodes that contain preaggregated records for data split after dimension removal, aggregating metrics for rows containing dimensions with identical values. These aggregated documents are then appended to the end of star-tree documents. If a document does contain a dimension with identical values, it traverses through the star node.
+Star nodes contain the aggregated data of all the other nodes for a particular dimension, acting as "catch-all" node. When a star node is found in a dimension, that dimension is skipped during aggregation. This group all values of that dimension together. This allows a query to skip non-competitive nodes when fetching the aggregated value of a particular field. 
 
-The star-tree index structure diagram contains the following three examples demonstrating how a document does or does not traverse star-tree nodes (indicated by the `*` symbol in the diagram) during a `Term` query, based on the average request size of the query and whether the document contains matching dimensions:
+The star-tree index structure diagram contains the following three examples demonstrating how a query behaves when retrieving aggregations from nodes in the star-tree:
 
-- When the port equals `8443` and the status equals `200`. Because the status equals `200`, the query does not traverse through a star node, and the aggregated metric is stored at the end of a star-tree document.
-- When the status equals `200`. The query traverses through a star node in the `port` dimension because `port` is not present as part of the query.
-- When the port equals `5600`. The query traverses through a star node in the `status` dimension because `status` is not present as part of the query.
+- **Blue**: In a `terms` query searching for the average request size aggregation, the `port` equals `8443` and the status equals `200`. Because the query contains values in both the `status` and `port` dimensions, the query traverses through staus node `200` and returns the aggregations from child node `8443`.
+- **Green**: In a `term` query searching for the count of requests aggregation, the `status` equals `200`. Because the query only contains a value from the `status` dimension, the query traverses to the `200` nodes child star node, which contains the aggregated vale of all the `port` children nodes.
```

Comment:
```suggestion
- **Green**: In a `term` query that searches for the number of requests aggregation, the `status` equals `200`. Because the query only contains a value from the `status` dimension, the query traverses the `200` node's child star node, which contains the aggregated value of all the `port` child nodes.
```